PicPi/MQTTManager.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MQTTManager.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 12/9/20.
+//
+
+import Foundation
+
+protocol MQTTManagerDelegate: AnyObject {
+  func manager(_ manager: MQTTManager, didUpdateConnectionStatus isConnected: Bool)
+ 
+}
+
+class MQTTManager {
```

Comment:
nit: how've i've learned to structure the class's vars and methods are like so:

```
class HomeViewController: UIViewController {
  private struct MyPrivateStructOrEnumOrOtherType {
  }

  let letOne = 2
  var varOne = false

  private let privateLet = 1
  private var privateVar = false

  init() {
  }

  deinit {
  }

  func myPublicMethod() {
  }

  // MARK: - Override UIViewController

  override func viewWillAppear(_ animated: Bool) {
  }

  // MARK: - UITableViewDataSource

  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
  }

  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
  }

  // MARK: - Private

  private func doSomethingPrivately() {
  }
}
```

I think this is not part of the swift style guild but a good rule of thumb to have consistent across swift files

Reply:
this is one of the thing I wanted to discuss in tonight meeting :)